test/dotnet-publish.Tests/GivenDotnetPublishPublishesProjects.cs

```diff
@@ -165,36 +183,25 @@ public void ItPublishesARidSpecificAppSettingSelfContainedToFalse()
                      .And.HaveStdOutContaining("Hello World");
         }
 
-        private DirectoryInfo PublishAppWithSelfContained(string testAppName, bool selfContained)
+        private DirectoryInfo PublishApp(string testAppName, string rid, string mode, string args = null)
         {
             var testInstance = TestAssets.Get(testAppName)
-                .CreateInstance($"PublishesSelfContained{selfContained}")
+                .CreateInstance($"PublishApp_{rid ?? "none"}_{mode ?? "none"}")
```

Comment:
Does `args` need to be included here as well?

Reply:
Pushed.